server/schedule/checker/replica_strategy.go

```diff
@@ -56,7 +56,7 @@ func (s *ReplicaStrategy) SelectStoreToAdd(coLocationStores []*core.StoreInfo, e
 		filter.NewExcludedFilter(s.checkerName, nil, s.region.GetStoreIds()),
 		filter.NewStorageThresholdFilter(s.checkerName),
 		filter.NewSpecialUseFilter(s.checkerName),
-		&filter.StoreStateFilter{ActionScope: s.checkerName, MoveRegion: true, AllowTemporaryStates: true},
+		&filter.LongTermStateFilter{ActionScope: s.checkerName, MoveRegion: true},
```

Comment:
Should it be TemporaryStateFilter?

Reply:
AllowTemporaryStates, if true, means only long-term status checks.